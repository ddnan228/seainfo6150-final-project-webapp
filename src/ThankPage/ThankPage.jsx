import React from 'react';
import css from "./ThankPage.module.css";

const ThankPage = () => {
    return (
        <div className={css.thankDiv}>
          <h1>Thank you for your valuable review!</h1>
        </div>
    )
}

export default ThankPage
