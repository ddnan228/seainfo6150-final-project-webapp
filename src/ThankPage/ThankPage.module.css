.thankDiv {
  text-align: center;
  padding-left: 22%;
  padding-right: 20%;
  color: #4d1f00;
  font-size: 25px;
  font-weight: bolder;
  font-family: 'Caveat', cursive;
}
