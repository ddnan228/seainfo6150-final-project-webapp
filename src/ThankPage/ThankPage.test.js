import React from "react";
import { render } from "@testing-library/react";
import ThankPage from "./ThankPage.jsx";

it("renders the ThankPage component correctly", () => {
  const { container } = render(<ThankPage />);
  expect(container).toMatchSnapshot();
});
