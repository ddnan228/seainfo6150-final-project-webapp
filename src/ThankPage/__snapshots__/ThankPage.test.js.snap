// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ThankPage component correctly 1`] = `
<div>
  <div
    class="thankDiv"
  >
    <h1>
      Thank you for your valuable review!
    </h1>
  </div>
</div>
`;
